app/views/full-page-examples/search/index.njk

```diff
@@ -81,24 +82,22 @@ notes: >-
           items: [
             {
               value: "hmrc",
-              text: "HM Revenue and Customs (HMRC)",
-              checked: values["organisation"]|length and "hmrc" in values["organisation"]
+              text: "HM Revenue and Customs (HMRC)"
             },
             {
               value: "employment-tribunal",
-              text: "Employment Tribunal",
-              checked: values["organisation"]|length and "employment-tribunal" in values["organisation"]
+              text: "Employment Tribunal"
             },
             {
               value: "mod",
-              text: "Ministry of Defence",
-              checked: values["organisation"]|length and "mod" in values["organisation"]
+              text: "Ministry of Defence"
             }
-          ]
+          ],
+          values: values["organisation"]
```

Comment:
@36degrees Your nice feature definitely saved a lot of code

* https://github.com/alphagov/govuk-frontend/pull/2616